cluster/evaluator/launch_eval.py

```diff
@@ -16,37 +16,34 @@
 sys.path.insert(0, '.')
 
 from absl import flags
-import jinja2
 import kubernetes
 import argparse
 import yaml
 import json
 import os
 import argh
 import fsdb
-import random
 import time
 
 
-def cross_eval(m1_path, m2_path, jobname, completions=20):
-    """
+def launch_eval_job(m1_path, m2_path, job_name, bucket_name="tensor-go-minigo-cross-evals",completions=20):
+    """Launches an evaluator job.
     m1_path, m2_path: full gs:// paths to the .pb files to match up
-    jobname: string, appended to the container, used to differentiate the job names
-    (e.g. 'minigo-gpu-evaluator-v5-123-v7-456')
+    job_name: string, appended to the container, used to differentiate the job names
+    (e.g. 'minigo-cc-evaluator-v5-123-v7-456')
+    bucket_name: Where to write the sgfs, passed into the job as $BUCKET_NAME
+    completions: the number of completions desired
     """
-    if m1_path is None or m2_path is None or jobname is None:
-        print("Provide all of m1_path, m2_path, and jobname params")
+    if m1_path is None or m2_path is None or job_name is None or bucket_name is None:
```

Comment:
consider:
all(var is not None for var in [m1_path, m2_path, job_name, bucket_name])
or
all([m1_path, m2_path, job_name, bucket_name])

Reply:
Done.